doc/ref_arch/kubernetes/chapters/chapter01.md

```diff
@@ -54,7 +120,7 @@ Figure 1-1 below shows this at a high level.
 <p align="center"><img src="../figures/ch01_k8s_arch.png" alt="Kubernetes Architecture for NFV" title="Kubernetes Architecture for NFV" width="100%"/></p>
 <p align="center"><b>Figure 1-1:</b> Kubernetes Architecture for NFV</p>
 
-### Other thoughts
+### More thoughts
 - Kubernetes / CaaS should be recognised in ETSI NFV / MANO
```

Comment:
IMO it should be made clear from the beginning that CaaS = Kubernetes + lots of backends behind the generic K8s APIs
the two are not synonyms, K8s in itself is not a CaaS IMO

Reply:
and CaaS is the thing which should be recognized as a VIM (or NFVI?) by ETSI

Reply:
Agree that CaaS = Kubernetes + other things.  I think we also need to define a **CaaS Manager** which is a management plane function that manages the lifecycle (instantiation, scaling, healing, etc.) of Kubernetes clusters, including communication with VIM for master/worker node lifecycle management.

Reply:
that would be the NFVO, right?
not necessarily specific for CaaS, IMO we need such orchestration/federation/POD LCM capabilities for all clouds, regardless type
I would say we simply need a CaaS-aware orchestrator

Reply:
It is important to differentiate between LCM for workloads executed on top of CaaS instance(s) (CNFs) - which should be the NFVO and the LCM for the CaaS instances themselves, which should be separated from the NFVI in the fuctional architecture (similarly to how it is described in the RM).

So NFVO manages the workloads, CaaS manager manages the CaaS (and/or?) IaaS instances.

Reply:
Yes, a CaaS is more than K8s. If you make the analogy to Linux, K8s is the kernel, but you need way more components in a Linux distribution.
We need to be more specific on what a "CaaS" is, though, because it's often misinterpreted as "platform to run containers" when in fact it also handles the full application lifecycle management --> architecturally, it sits _above_ the NFVI/VIM layer and goes up to the NFVO layer (it performs VNFM functionality).

Reply:
> So NFVO manages the workloads, CaaS manager manages the CaaS (and/or?) IaaS instances.

makes sense! only point I would add is that the two could be possibly collocated on the same physical entity in real life, even if we define them separately (seen such approach in open source, hence the comment).
and then maybe we should just call it "cloud manager"
and maybe it should manage multiple clouds not just one. especially because a big use-case of CaaS IMO will be the edge, and we shouldn't ask operators to manage 1000s of sites one-by-one (regardless if it is automated, or manual management)

Reply:
Agreed. There are pros and cons to both approaches. I would like to differentiate between "Infrastructure LCM" and "Workload LCM", because these functions exercise different APIs in different timeframes. Then, whether a given management system (implementation) deploys both together or separately should be a strictly implementation specific question.